tests/test_timefuncs.py

```diff
@@ -1035,3 +1035,28 @@ def test_timedelta_range() -> None:
 def test_dateoffset_freqstr() -> None:
     offset = DateOffset(minutes=10)
     check(assert_type(offset.freqstr, str), str)
+
+
+def timedelta64_and_arithmatic_operator() -> None:
+    s1 = pd.Series(data=pd.date_range("1/1/2020", "2/1/2020"))
+    s2 = pd.Series(data=pd.date_range("1/1/2021", "2/1/2021"))
+    check(assert_type((s1 - np.timedelta64(1, "M")), np.timedelta64), np.timedelta64)
+    check(assert_type((s1 + np.timedelta64(1, "M")), np.timedelta64), np.timedelta64)
+    check(assert_type(((s1 * np.timedelta64(1, "M")), np.timedelta64), np.timedelta64))
+    check(assert_type(((s1 / np.timedelta64(1, "M")), np.timedelta64), np.timedelta64))
```

Comment:
The results here are wrong, because `s1` is a `TimeStampSeries`, so adding and subtracting a `np.timedelta64` will produce the same.  Have to try each one out locally and see what the right type is.

Reply:
Sir it's worling fine with addition,substraction only problem is with multiplication,division any suggestions for that

Reply:
> Sir it's worling fine with addition,substraction only problem is with multiplication,division any suggestions for that

Doesn't seem that it is working.  Look at the log files from the CI runs (or run locally) and you can then see the error messages and fix the code.